### ISSUE TITLE
Logo proposal

### ISSUE THREAD

**Issue by jorlogicus** [picocms/Pico#499]
Greetings, I'm here today to offer my services. As a graphic designer in development and an open source enthusiast, I wish to contribute to this wonderful project by making a logo for it, the process (of course) is totally free and we would be working together to create the design that fits best with your project. Of course, you can quit whenever you want. In case you agree, I would like to ask you a few simple questions about your project to have a better understanding of it and so deliver a better product, also you could share with me any preferences you may have about colors, shapes, etc.

Sorry for disturbing if this is not the proper channel to request this, best regards.

**Comment by PhrozenByte** [comment]
Sure, why not, thank you @jorlogicus :+1:

However, be warned, I don't know anything about design :sweat_smile: Pico uses #2EAE9B as accent color and has a rather flat and simple design, aiming to convey some feeling of simplicity.

**Comment by jorlogicus** [comment]
I'll start right away and I'll be sure to make it as simple (minimalist) as possible. I'll be back with a few sketches as soon as I finish them.

**Comment by jorlogicus** [comment]
Here are some initial sketches, wanted to hear your opinion about which way should we follow.

![](https://cdn.discordapp.com/attachments/451449919648169984/601548972401754113/Recurso_1.png)

**Comment by PhrozenByte** [comment]
My overall first thought: They all look great! :heart:

Thank you @jorlogicus, great work! :+1: 

I guess I should simply share my second thoughts without much reflection, so please forgive me if my feedback sounds a bit weird... :speak_no_evil: 

I don't know why, but no. 3 heavily reminds me of a parrot :laughing:
No. 5 rather looks like a flag than a "P" - I like the idea of resembling a "P".
No. 2 closely follows the "simplicity" goal, however, for some reason it doesn't really "get" me.
I'm not sure where, but I think I've seen no. 4 somewhere else before - maybe some other pretty similar logo?
About no. 1 I honestly don't have much thoughts :expressionless: 

The logo should work on both very dark and very bright backgrounds, so even though the circle around no. 5 looks great, I fear it doesn't really meet this requirement. All other sketches work pretty well (since black and white as background are interchangeable). Rounded corners (like in no. 2, 4 and 5) IMHO look better. But I also like no. 2 with its full circle. Which is better depends on the contents, or what do you think?

Not sure about with which to go. I like no. 1, 2 and 4 a lot. I really would like to go with no. 2, because it perfectly meets Pico's simplicity claim. Maybe we can improve it further so that we can "fix" my second thought of "Hmm... It doesn't really get me"? But maybe this isn't really possible, so I really would like to keep no. 1 and 4 on the table.

So, let's talk about no. 2. I was thinking about having a white spot in the second bar (to resemble the "P" a bit better). Or maybe rounded "corners" for the third bar of the P (just like with no. 1 and 4)? Not sure about how this looks. I'm definitely no expert :sweat_smile:

**Some feedback from more people would be very appreciated!** :clap: The logo is for all Pico users.

**Comment by jorlogicus** [comment]
Perfect, thanks for the feedback. I'll come back as soon as I finish the new set of sketches. Sorry about the parrot-like logo, just came up with that due to "Pico" means Beak in Spanish.

**Comment by liviubarbu** [comment]
Hello, I've started using Pico for on of my project recently and i like it a lot for his simplicity and no database thing :). I'm visual designer and i want to jump in with one free logo proposal for pico cms. 
![pico_logo](https://user-images.githubusercontent.com/10975899/62255367-80305c80-b405-11e9-8475-69b65a2c79e4.jpg)

**Comment by jorlogicus** [comment]
Hi, I have some designs that I want you to see and give me your opinion about them.


![](https://cdn.discordapp.com/attachments/451449919648169984/603835198152376320/Recurso_18.png)

**Comment by PhrozenByte** [comment]
Sorry for the late response @liviubarbu and @jorlogicus. You two did a great job! :heart: 

Let's talk about @liviubarbu's proposal first. I really like the idea, it's super simple and still loosely resembles a "P" :+1:  However, I'm not sure whether it might be a bit too abstract. It instantly remembered me of a cloud callout (or a speech balloon, or a comments icon). Not sure, it doesn't really get me for a logo :thinking: 

About the small "pico" legend of the second form: I guess this won't really work most of the time. Just think of adding it to the header of both Pico's website and Pico's default theme - the legend's font size would be around just 8px or something.

I really like the idea putting Pico's claim right next to the logo! :+1: 

---

So many more proposals from @jorlogicus :smiley: I like all of them :+1: Let's try the same procedure as last time:

- For some (weird) reason no. 1 remembers me of Speedy Gonzales :laughing: I like the idea, it's "blazing fast" after all
- The rounding at the bottom of the P's arc on no. 5 feels a bit out of place. For some reason the [golden spiral](https://en.wikipedia.org/wiki/Golden_spiral) resp. the [Fibonacci spiral](https://en.wikipedia.org/wiki/Fibonacci_number) came to my mind to determine the rounding angles (1/φ for top-right, 1/φ² for right-bottom and 1/φ³ for bottom-left). Not sure whether this works...
- No. 2 still doesn't really get me. Maybe with a "hole" in the P's arc?
- Out of the blue there's no. 3. I really like it! :+1: Maybe with slightly (!) thicker strokes? Currently it feels a bit meek :smile:
- Last but not least there's no. 4. For some reason I don't have much thoughts about no. 4 :expressionless: The only thing I can say is that I tend to compare no. 4 with no. 1 and no. 1 "feels" better

In my opinion no. 5 has a great potential, but since I'm far away from being an expert here, my idea might also look totally awful :sweat_smile: So, idk, maybe we should try? I still like the idea of no. 2, maybe with said "hole" in the second bar to closer resemble a "P"? However, currently I'd say I prefer no. 3 the most. It's super simple, too and just "feels" better. Maybe just with slightly thicker strokes? No. 2, 3 and 5 are all very close together - both in how much I like them and their overall appearance resp. "style".

Besides no. 2, 3 and 5 I'd like to keep no. 1 on the table, too. The longer I view it the more I like it. I prefer no. 1 over both no. 4 of the second proposal and no. 1 and 4 of the first proposal.

Unfortunately I can't really give much reasoning about my opinions :unamused: So, once again, I really want to ask for more feedback. **Feedback from other people is highly appreciated!** :clap:

**Comment by TheTechRobo** [comment]
I’m no expert either but numbers 2, 3, and 4 are my personal favourites at first glance, but it’s tough :)

Really nice sketches! Can’t imagine me doing amazing ones like that 😃

**Comment by plingo** [comment]
> Hi, I have some designs that I want you to see and give me your opinion about them.
> 
> ![](https://camo.githubusercontent.com/e3667a8af8a4fb849a42a1b59c5c7448178a927e/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f6174746163686d656e74732f3435313434393931393634383136393938342f3630333833353139383135323337363332302f5265637572736f5f31382e706e67)

I like these:
![68747470733a2f2f63646e2e646973636f72646170702e636f6d2f6174746163686d656e74732f3435313434393931393634383136393938342f3630333833353139383135323337363332302f5265637572736f5f31382e706e67](https://user-images.githubusercontent.com/7942440/65700478-11c3ef80-e080-11e9-9e37-72cec47a9898.png)

**Comment by PhrozenByte** [comment]
Oh wow @type76, I think I've got a new favourite :smiley: Great work! It feels so "vivid", yet super simple. Incredible how just a small curvature can totally change the appearance. I really love it! As-is. There's only one thing: Could you please give it a try with Pico's accent color (`#2EAE9B`)?

**Comment by type76** [comment]
![pico-1](https://user-images.githubusercontent.com/20225335/67144678-2c4d4b00-f271-11e9-8f7f-ad8c6e776e35.png)

**Comment by type76** [comment]
Here's the SVG : https://io76.com/pico-logo/pico-1.svg

**Comment by PhrozenByte** [comment]
Thanks @type76! :heart: 

Before making a definite decision, I'd like to once again ask for feedback. *The following four logos are in the inner circle* and I'd like to ask everyone which they prefer. The new logo will be used starting with Pico 2.1 in both Pico's default theme and website, also see #515

[![pico-logo-selection](https://user-images.githubusercontent.com/920356/67160999-bcb18b80-f356-11e9-927e-bba49e7e5672.png)](https://user-images.githubusercontent.com/920356/67160999-bcb18b80-f356-11e9-927e-bba49e7e5672.png)

**Both feedback and votes are highly appreciated!** :heart:

**Comment by killerbees19** [comment]
3: Too much PayPal like 😂

**Comment by type76** [comment]
My list, in order goes; #1, #4, #2 and #3. 
Hot pink on black just seems sexiest to me, but I don't mind the green on black either.
Definitely use the black background.

If you need any design, let me know. I'd love to animate that #1 logo.

**Comment by Dric** [comment]
In order for me : 1, 4 (but I think the foot of the P is too long or too thin), 3 then 2.

**Comment by TheTechRobo** [comment]
2, 1, 3, 4 but it is SUPER tough. Great work both of you!

**Comment by type76** [comment]
[I animated it](https://io76.com/pico-logo/index.html) with a bit of css and added a tiny sound. Total overkill :)

**Comment by tumapav** [comment]
My order of preference would be 1,2,4,3.

I like the logo design versions that are in the Pico-green color theme and I prefer them over other color options. Other alternative is also Pico-green in the background and logo shape in white.

The hover animation by @type76 is a nice addition to the simple logo No. 1 and I wouldn't mind it if it would be used together with this logo for example on Pico's website. But currently it doesn't work on Firefox, it worked only when I opened it in Chrome.

**Comment by PhrozenByte** [comment]
Many thanks for all your feedback and a big :clap: :clap: :clap: for @type76 and his logo suggestion - **we finally got an official logo for Pico!** :confetti_ball: :tada:

A picture is worth a thousand words and we can finally convey Pico’s idea of creating websites using a single picture: Stupidly simple and blazing fast, making the web easy.

@type76 you did an amazing job here, I think I can speak for the whole community (the votes are pretty clear), we absolutely love it! :heart_eyes: Thank you! :heartbeat: :heart: :heartbeat:

Another big thank you to @jorlogicus for bringing this up and your absolutely stunning logo suggestions! The same is true for @liviubarbu, thank you for your great suggestion, too!  Even though we chose @type76's suggestion in the end, it's very important to make one thing clear: You guys did an amazing job! :heart: Another :clap: :clap: :clap: for you guys :+1: 

The logo will be included starting with Pico 2.1 (see #515)

---

Here's how it is going to look like in Pico's default theme starting from Pico 2.1 (now also including Pico's tagline; also see [picocms/pico-theme#2](https://github.com/picocms/pico-theme/pull/2)):

![pico-21](https://user-images.githubusercontent.com/920356/69486985-b3827580-0e52-11ea-98e1-0fa02c61d952.png)

---

If you didn't follow this thread from the beginning, this is @type76's suggestion and now Pico's official logo (also available as SVG, will be available at http://picocms.org/logo.svg later):

![logo](https://user-images.githubusercontent.com/920356/69486660-9055c700-0e4e-11ea-8c1a-4a2498827edb.png)
